builtin/mainmenu/dlg_config_world.lua

```diff
@@ -160,19 +160,41 @@ local function handle_buttons(this, fields)
 		local rawlist = this.data.list:get_raw_list()
 
 		local i,mod
+		-- mod_decisions helps keeping track of what value to assign to the
+		-- load_mod setting, as it only allows one
+		-- (nil = not yet decided, otherwise the value of mod.enabled for the
+		-- mod that was chosen)
+		local mod_decisions = {}
+		local conflict_warning, conflict_list
 		for i,mod in ipairs(rawlist) do
+			local load_mod_setting = "load_mod_"..mod.name
 			if not mod.is_modpack and
 					not mod.is_game_content then
 				if modname_valid(mod.name) then
-					worldfile:set("load_mod_"..mod.name, tostring(mod.enabled))
+					local decision = mod_decisions[mod.name]
+					if decision == nil or decision == false and mod.enabled then
+						mod_decisions[mod.name] = mod.enabled
+						worldfile:set(load_mod_setting, mod.enabled or "false")
+					elseif decision and mod.enabled then
+						if conflict_warning then
+							conflict_list = conflict_list .. ", \""
+						else
+							conflict_warning = "Multiple mods with the same name were chosen for the following names: $1"
```

Comment:
Not detected by our translation script. Please cache all translated strings (`fgettext_ne("text here $1 foo bar")` in variables and assign `conflict_warning` to the matching one.